### PR TITLE
Update to upstream 7.0.0-beta-3.0

### PR DESCRIPTION
**Pull request by amisevsk**
### What does this PR do?
Updates upstream dependency to 7.0.0-beta-3.0 and makes necessary changes to `pom.xml`s

Depends on https://github.com/redhat-developer/rh-che/pull/1362

### CODE REVIEW

File: assembly/assembly-wsmaster-war/pom.xml [hunk 1]
```diff
@@ -87,6 +87,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.che.core</groupId>
+            <artifactId>che-core-api-auth-github</artifactId>
+            <version>7.0.0-beta-3.0</version>
```
Comment (ibuziuk) [inline assembly/assembly-wsmaster-war/pom.xml]: dep version should be removed

Comment (amisevsk) [reply assembly/assembly-wsmaster-war/pom.xml]: Right, good catch; it was a copy paste from maven

Fixed.

**Comment by amisevsk** [comment]
Looks to be a functional-tests failure, @Katka92 could you take a look when you have a moment?

`Che7DialogAbout.createAndOpenWorkspace:59->createChe7Workspace:84->prepareWorkspace:95 » Timeout`

Depending on which workspace this is, I noticed an issue starting a theia:next workspace yesterday, so it may be that.

**Comment by amisevsk** [comment]
[test]

**Comment by Katka92** [comment]
@amisevsk This is weird, I can not reproduce this with my own account by manually visiting dashboard. I'll take a look and write my findings here.

**Comment by Katka92** [comment]
Yep, so prod and prod-preview works correctly. This is manually reproducible on rh-che deployment with new version. Sorry for confusion.

**Comment by ibuziuk** [comment]
> Nevermind, dashboard appears to be totally broken.

Yeah, looks like dashboard is not rendered properly 

[image](https://user-images.githubusercontent.com/1461122/56189431-3ffcc800-6028-11e9-9796-682a712a7e2b.png)

**Comment by ibuziuk** [comment]
[test]

**Comment by amisevsk** [comment]
Yeah I don't know what's going on -- I built it locally and it works without issue.

**Comment by Katka92** [comment]
@amisevsk I can try to build it locally too if you want.

**Comment by ibuziuk** [comment]
@Katka92 yes, if possible, please try the manual build of this PR. It is quite a weird issue with PR check and UD problems

**Comment by ibuziuk** [comment]
[test]

**Comment by amisevsk** [comment]
Seems that https://github.com/eclipse/che/pull/13033 is what's causing the issue, since we overwrite the `create-workspace.controller.ts` file. In retrospect this would've been fairly easy to see by looking at the automatically generated source diff (`./target/src_tree.diff`), so that's good to know for next time.

**Comment by ibuziuk** [comment]
@amisevsk great job!

**Pull request merged**